dask_sql/mappings.py

```diff
@@ -107,6 +107,18 @@ def python_to_sql_type(python_type) -> "DaskTypeMap":
             tz=str(python_type.tz),
         )
 
+    try:
+        from cudf.api.types import is_decimal_dtype
+
+        if is_decimal_dtype(python_type):
```

Comment:
Wonder if there's a way we could implement this function internally in a similar fashion to `is_cudf_type` to avoid the cuDF import here?